schematic_api/api/openapi/api.yaml

```diff
@@ -709,6 +709,16 @@ paths:
           description: ID of view listing all project data assets. For example, for Synapse this would be the Synapse ID of the fileview listing all data assets for a given project.(i.e. master_fileview_id in config_example.yml)
           example: syn23643253
           required: true
+        - in: query
+          name: project_scope
+          schema:
+            type: array
+            items: 
+              type: string
+            nullable: false
+          description: List, a subset of the projects contained within the asset view that are relevant for the current operation. Speeds up some operations that interact with Synapse.
```

Comment:
Please change the defaults so a result will be returned to give a user an idea what to expect. Also please update the description to reflect that in this case for most people the `project_scope` will be the same as the `project_id`

Reply:
Thanks @mialy-defelice! I just updated the default example in `api.yml` and removed the project_scope parameter there too. I think when I was testing, I happened to use a folder ID instead of a project ID so that I thought that it didn't work. But just like you said, we shouldn't need the project_scope parameter since we could code project_id directly as project_scope for users.